lab3/client/src/views/generator.js

```diff
@@ -0,0 +1,30 @@
+export default function (store) {
+	return class GeneratorComponent extends window.HTMLElement {
+		constructor () {
+			super();
+			this.store = store;
+
+			// TODO: render generator initial view
+				this.root.innerHTML = `<ul>
+				${this.store.subscribe()}
+				</ul>`
```

Comment:
This is not how generator should have as HTML.

The subscribe function doesn't return anything.